src/js/containers/PublicRecord.js

```diff
@@ -1,152 +1,161 @@
+          <button className='btn btn-primary' onClick={this.handleArchiveUrl.bind(this, query)}>Archive Url</button>
         </div>
-      );
+      )
     }
 
-    return undefined;
+    return undefined
   }
 
-  renderSearchResults() {
+  renderSearchResults () {
     return (
-      <div className="row">
-        <div className="col-md-12">
-          <label className="label">results</label>
+      <div>
+        <div className='row'>
+          <div className='col-md-10 offset-md-1'>
+            <label className='label'>results</label>
+            <div className='row'>
+              <List component={SearchResultItem} data={this.props.results} />
+              {this.renderArchiveUrl()}
+            </div>
+          </div>
         </div>
-        <List component={SearchResultItem} data={this.props.results} />
-        {this.renderArchiveUrl()}
       </div>
-    );
+    )
   }
 
-  renderRecentContent() {
+  renderRecentContent () {
     return (
       <div>
-        <div className="row">
-          <div className="col-md-12">
+        {/* <div className='row'>
+          <div className='col-md-10 offset-md-1'>
             <hr />
-            <label className="label">Sources</label>
+            <label className='label'>Collections</label>
+            <div className='row'>
+              <List data={this.props.collections} component={CollectionItem} />
+            </div>
           </div>
-          <List data={this.props.sources} component={SourceItem}  />
-        </div>
-        <div className="row">
-          <div className="col-md-12">
+        </div> */}
+        <div className='row'>
+          <div className='col-md-10 offset-md-1'>
             <hr />
-            <label className="label">New Content Needing Metadata:</label>
+            <label className='label'>New Content Needing Metadata:</label>
+            <div className='row'>
+              <List component={ContentItem} data={this.props.recentContent} />
+            </div>
           </div>
-          <List component={ContentItem} data={this.props.recentContent} />
         </div>
       </div>
-    );
+    )
   }
 
-  render() {
-    const { query, results } = this.props;
+  render () {
+    const { query, results } = this.props
 
     return (
-      <div id="public-record" className="public-record page">
+      <div id='public-record' className='public-record page'>
         <header>
-          <div className="container">
-            <div className="row">
-              <div className="col-md-12">
+          <div className='container'>
+            <div className='row'>
+              <div className='col-md-10 offset-md-1'>
+                <hr />
+                <h3>Hi there!</h3>
+                <p>This is an early, experimental version of the data together platform. If you'd like more info, be sure to check out <a href='https://datatogether.org'>datatogether.org</a></p>
```

Comment:
Update language to be inline with comment from @flyingzumwalt on mission?
(maybe we just handle that as a new PR when that discussion finishes)